.github/azure-steps.yml

```diff
@@ -111,7 +111,7 @@ steps:
     condition: eq(variables['python_version'], '3.8')
 
   - script: |
-      ${{ parameters.prefix }} python -m pip install thinc-apple-ops
+      ${{ parameters.prefix }} python -m pip install "thinc-apple-ops>=0.1.0.dev0"
```

Comment:
The only slightly more general option might be to use `--pre` here instead?

Reply:
I didn't know about that option. So, then this should be

```
      ${{ parameters.prefix }} python -m pip install --pre "thinc-apple-ops>=0.1.0"
```

?

Reply:
Just

```
python -m pip install --pre thinc-apple-ops
```

Reply:
I'm not sure if we'd have cases where we specifically wouldn't want `--pre`, but it's probably okay as a default here for this test?

Reply:
Yeah, I think for CI this should be completely safe. I've added a commit to use `--pre`.